csp/utils.py

```diff
@@ -53,9 +61,20 @@ def build_policy(config=None, update=None, replace=None):
                 csp[k] += tuple(v)
 
     report_uri = csp.pop('report-uri', None)
+    upgrade_insecure_requests = csp.pop('upgrade-insecure-requests', None)
+    block_all_mixed_content = csp.pop('block-all-mixed-content', None)
+
     policy = ['%s %s' % (kk, ' '.join(vv)) for kk, vv in
               csp.items() if vv is not None]
+
     if report_uri:
         report_uri = map(force_text, report_uri)
         policy.append('report-uri %s' % ' '.join(report_uri))
+
+    if upgrade_insecure_requests:
+        policy.append('upgrade-insecure-requests ')
+
+    if block_all_mixed_content:
+        policy.append('block-all-mixed-content ')
```

Comment:
Instead of special casing `upgrade-insecure-requests` and `block-all-mixed-content`, what do you think about changing the policy iterator above (`policy = ['%s %s' ...`) to make items with a value of `True` work like these items need to? Something like

```python
policy_parts = []

for key, value in csp.items():
    if value is True:
        policy_parts.append(key)
    elif value is False:
        pass
    else:
        policy.append('%s %s' % (key, ' '.join(value))
```

Additionally, I don't think that a space is needed at the end here, if we decide to keep it.